docs/guides/expectations/creating_custom_expectations/overview.md

```diff
@@ -1,18 +1,76 @@
 ---
-title: Creating custom Expectations
+title: Overview
 ---
-import Prerequisites from '../../connecting_to_your_data/components/prerequisites.jsx'
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import Prerequisites from './components/prerequisites.jsx'
 
+You can extend the functionality of Great Expectations by creating your own custom Expectations. You can also enrich Great Expectations as a shared standard for data quality by contributing new Expectations to the open source project.
 
-(Text goes here)
+Both processes follow the same streamlined steps. This section will teach you how.
 
-<Prerequisites>
+<Prerequisites></Prerequisites>
 
-- [Set up a working deployment of Great Expectations](../../../tutorials/getting_started/intro.md)
-OR
-- [Set up a working deployment of Great Expectations](../../../tutorials/getting_started/intro.md)
-    
-</Prerequisites>
+## Steps to create a custom Expectation
 
+A fully developed Expectation needs to do a lot of things:
+* Execute consistently across many types of data infrastructure
+* Render itself and its Validation Results into several different formats
+* Support Profiling against new data
+* Be maintainable, with good tests, documentation, linting, type hints, etc.
+
+In order to make development of Expectations as easy as possible, we've broken up the steps to create custom Expectations into a series of bite-sized steps. Each step can be completed in minutes. They can be completed (and contributed) incrementally, unlocking value at each step along the way.
+
+Grouped together, they constitute a Definition of Done for Expectations at each [Level of Maturity](/docs/contributing/contributing_maturity).
+
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+<i class="fas fa-circle" style={{color: "#dc3545"}}></i> An experimental Expectation...
+
+* Has a `library_metadata` object
+* Has a docstring, including a one-line short description
+* Has at least one positive and negative example case
+* Has core logic and passes tests on at least one `Execution Engine`
+
+<i class="fas fa-circle" style={{color: "#ffc107"}}></i> A beta Expectation...
+
+* Has basic input validation and type checking
+* Has all four statement `Renderers`: question, descriptive, prescriptive, diagnostic
+* Has default `Parameter Builders` and Domain hooks to support Profiling   
+* Has core logic exists and passes tests for all applicable `Execution Engines` and SQL dialects
```

Comment:
"Has core logic exists and passes" reads as though a couple of variations of the statement were composed and part of the losing variation wasn't removed.  I'd remove the word 'exists.'

Reply:
ha. You're totally right.
```suggestion
* Has core logic that passes tests for all applicable `Execution Engines` and SQL dialects
```